### ISSUE TITLE
ultrachord sample integration

### ISSUE THREAD

**Issue by langel** [langel/botb_irc_bot#34]
got the old bot zip from strobe
added the old ultrachord samples
they have different ranges
most are one octave (so they should ignore the octave parameter?)
choirs and trem cover 5 octaves which is a lesser range than sox gen instruments

**Comment by langel** [comment]
I was talking with Yuki on this yesterday.  They suggested that the samples not be held inside our repository.  It would be pretty simple for me to host a zip on BotB that the bot can donload and unpack if the sample directory is empty.  I don't really want to go crazy with the number of instruments; can be up to our discretion.  Seems like Strobe's files already follow a naming convention.  I know I'd love to have some Roland D50 pizzagogo samples!

**Comment by jangler** [comment]
I agree with ~~not~~ having the samples being stored elsewhere. If the sample files have a consistent naming convention, I think it should be pretty straightforward for me to do this with SoX without any need for a ton of enumeration code.
